cupy/testing/helper.py

```diff
@@ -492,7 +492,8 @@ def check_func(x, y):
 
 
 def numpy_cupy_array_list_equal(
-        err_msg='', verbose=True, name='xp', sp_name=None, scipy_name=None):
+        err_msg='', verbose=True, name='xp',
+        sp_name=None, scipy_name=None, type_check=True):
```

Comment:
IMO, the deprecation of the function does not need the change of its interface while it may be reasonable to add `type_check` argument for consistency.

We may write `type_check = False` inside the function to clarify the difference between the function and `numpy_cupy_array_equal`.

Reply:
Yeah, you are absolutely right.